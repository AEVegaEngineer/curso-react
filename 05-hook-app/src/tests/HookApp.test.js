import React from 'react' 
import { shallow } from 'enzyme'
import { HookApp } from '../HookApp'

describe('Pruebas de HookApp', () => {
    let wrapper = shallow(<HookApp/>);

    beforeEach( () => {
        wrapper = shallow(<HookApp/>);
    });

    test('Debe mostrar el snapshot del componente correctamente', () => {
        wrapper = shallow(<HookApp/>);
        expect(wrapper).toMatchSnapshot();
    })
    
});