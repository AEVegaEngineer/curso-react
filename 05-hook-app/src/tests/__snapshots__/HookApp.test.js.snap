// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de HookApp Debe mostrar el snapshot del componente correctamente 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
